cmd/fyne/internal/commands/build.go

```diff
@@ -7,16 +7,72 @@ import (
 	"strings"
 
 	version "github.com/mcuadros/go-version"
+	"github.com/urfave/cli/v2"
 )
 
-type builder struct {
+// Builder generate the executables.
+type Builder struct {
 	os, srcdir, target string
 	release            bool
 	tags               []string
+	tagsToParse        string
 
 	runner runner
 }
 
+// Build returns the cli command for building fyne applications
+func Build() *cli.Command {
+	b := &Builder{}
+
+	return &cli.Command{
+		Name:        "build",
+		Usage:       "Build an application.",
+		Description: "You may specify the --executable to build, but --target will define the OS it is build for indiscriminate of the extention of the executable.",
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:        "target",
+				Aliases:     []string{"os"},
+				Usage:       "The mobile platform to target (android, android/arm, android/arm64, android/amd64, android/386, ios, iossimulator).",
+				Destination: &b.os,
+			},
+			&cli.StringFlag{
+				Name:        "sourceDir",
+				Aliases:     []string{"src"},
+				Usage:       "The directory to package, if executable is not set.",
+				Destination: &b.srcdir,
+			},
```

Comment:
How about dropping this and instead using the command argument as input? This would make the command more like `go build ./cmd/fyne_demo` and we can avoid the need to remember the flag.

Reply:
Good point. We can do so.

Reply:
> How about dropping this and instead using the command argument as input? This would make the command more like `go build ./cmd/fyne_demo` and we can avoid the need to remember the flag.

Source dir and target package are not the same thing though... I think the go parameters take a package as parameter not directory.
Perhaps we need to move all our commands over time? "fyne get" is already there